walsamlee/politico
# #163690223 Users Signup

#### What does this PR do?
Add `signup` to consume the `signup` endpoint on the `UI`
#### Description of Task to be completed?
Have user `signup page` on the `UI` working
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-users-signup-163690223`
- Visit `https://walsamlee.github.io/politico/users/signup.html` on the browser
#### What are the relevant pivotal tracker stories?
#163690223
